test/test_with_xpress.jl

```diff
@@ -1549,9 +1549,9 @@ end
     input_data["DomesticHotWaterLoad"]["doe_reference_name"] = building
     elec_load = REopt.ElectricLoad(latitude=latitude, longitude=longitude, doe_reference_name=building)
     input_data["ElectricLoad"]["annual_kwh"] = elec_load_multiplier * sum(elec_load.loads_kw)
-    space_load = REopt.SpaceHeatingLoad(latitude=latitude, longitude=longitude, doe_reference_name=building, existing_boiler_efficiency=input_data["ExistingBoiler"]["efficiency"])
+    space_load = REopt.HeatingLoad(load_type="space_heating", latitude=latitude, longitude=longitude, doe_reference_name=building, existing_boiler_efficiency=input_data["ExistingBoiler"]["efficiency"])
     input_data["SpaceHeatingLoad"]["annual_mmbtu"] = heat_load_multiplier * space_load.annual_mmbtu / input_data["ExistingBoiler"]["efficiency"]
-    dhw_load = REopt.DomesticHotWaterLoad(latitude=latitude, longitude=longitude, doe_reference_name=building, existing_boiler_efficiency=input_data["ExistingBoiler"]["efficiency"])
```

Comment:
It's good that we're keeping up with this, but is it worth a conversation revisiting whether we keep these?  I think that with battery degaradation updated, all indicator constraints have been refactored for HiGHS compatibility.

Reply:
Right, I agree we could remove "test_with_xpress.jl". Let's punt on that for now though.